### ISSUE TITLE
Edited messages are not updated into the aggregated channel

### ISSUE THREAD

**Issue by tomodachi94** [Astralchroma/Aggregator#39]
Hello! This is a very useful bot you've made.

Unfortunately, when a message is edited in the original server, the edit is not propagated to the aggregated channel.

Is this the intended functionality?

Thank you for making this!

**Comment by Astralchroma** [comment]
No, it isn't intended functionality. The issue was already known, we just haven't gotten around to fixing it. We have been intending to do a Rust rewrite at some point, so unless someone PRs a fix, it probably won't be fixed until then.

**Comment by tomodachi94** [comment]
Okay thank you!
